packages/brand-positioning-block/src/components/Item.tsx

```diff
@@ -0,0 +1,40 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { joinClassNames } from '@frontify/guideline-blocks-shared';
+import { useDraggable } from '@dnd-kit/core';
+import { ItemProps } from '../types';
+import { Toolbar } from './Toolbar';
+
+export const Item = ({ src, x, y, id }: ItemProps) => {
+    const editing = true;
+    const { attributes, isDragging, listeners, setNodeRef, transform } = useDraggable({
+        id,
+    });
+    const transformStyle = {
+        transform: transform ? `translate(${transform.x}px, ${transform.y}px)` : '',
+        zIndex: isDragging ? 2 : 1,
+    };
+
+    return (
+        <div
+            style={{
+                left: `${x}%`,
+                top: `${y}%`,
+                ...transformStyle,
+            }}
+            ref={setNodeRef}
+            className="tw-h-[50px] tw-w-[50px] tw-bg-white tw-group tw-rounded tw-border tw-border-line tw-p-1 tw-absolute hover:tw-outline-offset-[1px] hover:tw-outline hover:tw-outline-[1px] hover:tw-outline-box-selected-inverse"
```

Comment:
I'd add another div and use `border` again instead of `outline`, but that's up for discussion.

Reply:
Hmmmm.. Thing is that would increase the width and height of the component a bit and that is not always desired. Outline can also potentially show over other elements giving it a nice highlighting. What do you think?